--- Makefile.bak	2024-01-16 14:48:14.786420851 +0700
+++ Makefile	2024-01-16 15:05:42.669258607 +0700
@@ -21,11 +21,13 @@
 DEFINES = -DUSE_FEEDING_METHOD_BUFFER
 INCLUDES = -I./include/khronos -I./core -I./component/video/wave5/common -I./component/video/wave4 -I./component/helper
 $(warning "the value of LOCAL_PATH is$(STAGING_DIR)")
-INCLUDES += -I$(STAGING_DIR)/usr/include/
-INCLUDES += -I$(STAGING_DIR)/usr/include/wave420l/
-INCLUDES += -I$(STAGING_DIR)/usr/include/wave420l/vpuapi/
-INCLUDES += -I$(STAGING_DIR)/usr/include/wave420l/sample/helper/
-#INCLUDES += -I$(STAGING_DIR)/usr/include/wave420l/sample/helper/misc/
+INCLUDES += -I..
+INCLUDES += -I../wave420l/
+INCLUDES += -I../wave420l/code/
+INCLUDES += -I../wave420l/vpuapi/
+INCLUDES += -I../wave420l/code/vpuapi/
+INCLUDES += -I../wave420l/sample/helper/
+INCLUDES += -I../wave420l/sample/helper/misc/
 
 #mjpeg
 INCLUDES += -I./component/image/common
@@ -100,4 +102,4 @@
 	-mkdir -p $(OBJDIR)
 
 obj/%.o: %.c $(MAKEFILE)
-	$(CC) $(CFLAGS) -Wall -Werror -c $< -o $@ -MD -MF $(@:.o=.dep)
+	$(CC) $(CFLAGS) -Wall -Werror -Wno-sometimes-uninitialized -Wno-enum-conversion -c $< -o $@ -MD -MF $(@:.o=.dep)
